lib/fog/libvirt/requests/compute/list_volumes.rb

```diff
@@ -31,7 +31,7 @@ def volume_to_attributes(vol)
             :format_type => format_type,
             :allocation  => bytes_to_gb(vol.info.allocation),
             :capacity    => bytes_to_gb(vol.info.capacity),
-          }
+          } rescue nil # If there are issues during stat of volume file
```

Comment:
This is related to issue #13 .
I don't know that much Ruby to pinpoint the exception that's being raised.
If you can name the exception, I can change the code easily to comply.